src/features/firstLogin/tutorials/components/GenericCard.test.tsx

```diff
@@ -0,0 +1,133 @@
+import { renderHook } from '@testing-library/react-hooks'
+import { fireEvent, render } from '@testing-library/react-native'
+import AnimatedLottieView from 'lottie-react-native'
+import React, { RefObject } from 'react'
+import { View } from 'react-native'
+import * as Animatable from 'react-native-animatable'
+
+import { analytics } from 'libs/analytics'
+import GeolocationAnimation from 'ui/animations/geolocalisation.json'
+
+import {
+  CardProps,
+  GenericCard,
+  useAnalyticsLogScreenView,
+  useButtonAnimation,
+  usePlayAnimation,
+} from './GenericCard'
+
+describe('<GenericCard />', () => {
+  const animation = GeolocationAnimation
+  const buttonCallback = jest.fn()
+  const buttonText = 'Submit'
+  const pauseAnimationOnRenderAtFrame = 62
+  const subTitle = 'Subtitle'
+  const text = 'Text'
+  const title = 'Title'
+  it('should render correctly', async () => {
+    const { getByText } = renderGenericCardComponent({
+      buttonText,
+      animation,
+      buttonCallback,
+      pauseAnimationOnRenderAtFrame,
+      subTitle,
+      text,
+      title,
+    })
+    expect(await getByText(title)).toBeTruthy()
```

Comment:
getByText ne renvoie pas de promise : 
```suggestion
    expect(getByText(title)).toBeTruthy()
```

Reply:
Tu peux enlever l'async du test du coup